tdrs-backend/tdpservice/settings/common.py

```diff
@@ -152,7 +152,7 @@ class Common(Configuration):
         }
 
     # General
-    APPEND_SLASH = False
+    APPEND_SLASH = True
```

Comment:
This prevents a 404 error from being raised when the trailing slash isn't included at the end of an API URL. Instead it will redirect to add the slash that is needed. Note however that this redirect causes a CORS preflight error on the frontend so we still need to ensure that the trailing slash is present in all UI code.